.gitignore

```diff
@@ -33,7 +33,8 @@ bower_components
 build/Release
 
 # Dependency directories
-node_modules/
+client/node_modules/
+sever/node_modules/
```

Comment:
I guess just `node_modules/` is enough?

Reply:
Yes, but I believe adding `node_modules/` in `.gitignore` means ignoring all directories called `node_modules` in current folder and any subfolders. So both `client/node_modules/` and `sever/node_modules/` will be ignored.

Reply:
Oh, I see. Learnt this from here and will change it back. Thank you Max!